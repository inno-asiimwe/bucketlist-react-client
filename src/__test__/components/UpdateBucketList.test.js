import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import UpdateBucketlistForm from '../../components/UpdateBucketList';

describe('UpdateBucketlistForm', () => {
  it('Matches a snapshot', () => {
    const output = shallow(<UpdateBucketlistForm />)
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
