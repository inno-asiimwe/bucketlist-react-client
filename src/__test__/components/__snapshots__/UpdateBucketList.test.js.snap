// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpdateBucketlistForm Matches a snapshot 1`] = `
<Connect(Form(UpdateBucketlistForm))
  destroyOnUnmount={true}
  enableReinitialize={false}
  forceUnregisterOnUnmount={false}
  form="UpdateBucketlistForm"
  getFormState={[Function]}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  pure={true}
  shouldAsyncValidate={[Function]}
  shouldError={[Function]}
  shouldValidate={[Function]}
  shouldWarn={[Function]}
  touchOnBlur={true}
  touchOnChange={false}
  updateUnregisteredFields={false}
  validate={[Function]}
/>
`;
